helpdesk_factorlibre/models/inherit_res_partner.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# © 2020 Nacho Morales
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import models, fields, _, api
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+    
+    
+    helpdesk_ticket_ids = fields.One2many(
+        string='Tickets',
+        comodel_name='helpdesk.ticket',
+        inverse_name='partner_id',
+    )
+    
+    count_tickets = fields.Integer(
+        string='Number of tickets',
+        compute='_compute_count_tickets'
+    )
+    
+    @api.multi
+    def _compute_count_tickets(self):
+        for record in self:
+            #Codigo mas eficiente: record.count_tickets = len(record.helpdesk_ticket_ids)
```

Comment:
this comment for example...
#REVIEW: use "record.count_tickets = len(record.helpdesk_ticket_ids)" to efficient code

Reply:
Fix in 31c0c3d9567a5b7680d44351437cb1eacb729ef6